api/api.go

```diff
@@ -226,6 +226,11 @@ func (api *ExtensionsApi) Notify(extensions []core.Extension) {
 			if err != nil {
 				log.Printf("failed to merge update data %v", err)
 			}
+			// manually overwite localization data
+			if castedData.Development.LocalizationStatus == "success" {
+				api.Extensions[index].Localization = castedData.Localization
+				api.Extensions[index].Localization.LastUpdated = time.Now().Unix()
```

Comment:
Can we update the timestamp inside `GetLocalization` instead? I'm going to remove the code above to update the build script's `LastUpdated` outside of the API in the next PR. Given that many things can send a notification, we don't want to update to modify the last update timestamp here as it won't be accurate.